index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Evans_Portfolio</title>
+    <script
+      src="https://kit.fontawesome.com/0ec53d8e70.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header id="navigation">
+      <div class="logo">
+        <h3>Evans</h3>
+      </div>
+      <nav class="navlinks">
+        <ul class="links">
+          <li class="link"><a href="#portfolio">Portfolio</a></li>
+          <li class="link"><a href="#about">About</a></li>
+          <li class="link"><a href="#contact">Contact</a></li>
+          <li>
+            <a href="#"><i class="fa-regular fa-envelope"></i></a>
+          </li>
+        </ul>
+      </nav>
+      <div class="toggle">
+        <div class="menu"></div>
+      </div>
+    </header>
+
+    <section id="hero">
+      <div class="hero_text">
+        <h1 class="title">
+          Hey There <br />
```

Comment:
- [ ] Please, kindly add dots to this and the next, to follow the design suit. 😀